packages/anoncreds/src/utils/w3cAnonCredsUtils.ts

```diff
@@ -199,7 +200,7 @@ export function getW3cRecordAnonCredsTags(options: {
     anonCredsMethodName: methodName,
     anonCredsRevocationRegistryId: revocationRegistryId,
     anonCredsCredentialRevocationId: credentialRevocationId,
-    ...(isIndyDid(issuerId) && {
+    ...((isIndyDid(issuerId) || isUnqualifiedIndyDid(issuerId)) && {
```

Comment:
I've understood from a previous comment that it was better to store the qualified form because it was not possible to go the other way around. Now I'm confused!